types/systemjs/index.d.ts

```diff
@@ -1,388 +1,103 @@
-         */
-        isModule(object: any): boolean;
-
-        /**
-         * Given a plain JavaScript object, return an equivalent Module object.
-         * Useful when writing a custom instantiate hook or using System.set.
-         */
-        newModule(object: any): any;
-
-        /**
-         * Declaration function for defining modules of the System.register polyfill module format.
-         */
-        register(name: string, deps: string[], declare: (...modules: any[]) => any): void;
-        register(deps: string[], declare: (...modules: any[]) => any): void;
-
-        /**
-         * Companion module format to System.register for non-ES6 modules.
-         * Provides a <script>-injection-compatible module format that any CommonJS or Global module can be converted into for CSP compatibility.
-         */
-        registerDynamic(name: string, deps: string[], executingRequire: boolean, declare: (...modules: any[]) => any): void;
-        registerDynamic(deps: string[], executingRequire: boolean, declare: (...modules: any[]) => any): void;
-
-        /**
-         * Sets a module into the registry directly and synchronously.
-         * Typically used along with System.newModule to create a valid Module object:
-         */
-        set(moduleName: string, module: any): void;
-
-        /**
-         * Resolves module name to normalized URL.
-         */
-        resolve(moduleName: string, parentName?: string): Promise<string>;
-
-        /**
-         * Resolves module name to normalized URL.
-         * Synchronous alternative to `SystemJS.resolve`.
-         */
-        resolveSync(moduleName: string, parentName?: string): string;
-
-        /**
-         * In CommonJS environments, SystemJS will substitute the global require as needed by the module format being
-         * loaded to ensure the correct detection paths in loaded code.
-         * The CommonJS require can be recovered within these modules from System._nodeRequire.
-         */
-        _nodeRequire(dep: string): any;
-
-        /**
-         * Modules list available only with trace=true
-         */
-        loads: PackageList<any>;
-
-        env: string;
-
-        loaderErrorStack: boolean;
-
-        packageConfigPaths: string[];
-
-        /**
-         * Specify a value of true to have SystemJS conform to the AMD-style plugin syntax, e.g. "text!some/file.txt", over the default of "some/file.txt!text".
-         */
-        pluginFirst: boolean;
-
-        version: string;
-
-        /**
-         * Enables the output of warnings to the console, including deprecation messages.
-         */
-        warnings: boolean;
-    }
+// TypeScript Version: 3.5
+
+declare const System: {
+  /**
+   * Loads a javascript module from either a url or bare specifier that is in an import map.
+   * You may optionally provide a parentUrl that will be used for resolving relative urls.
+   */
+  // tslint:disable-next-line no-unnecessary-generics
```

Comment:
This suppression seems unused here? right?

Reply:
Yep good catch. Just pushed a commit that removes that